src/lib/pages/workflows-with-new-search.svelte

```diff
@@ -79,6 +79,12 @@
     }
   });
 
+  afterUpdate(() => {
+    if (!query && ($workflowFilters.length || $workflowSorts.length)) {
```

Comment:
> either we need to clear all filters on Workflow nav click, or apply them to the query

@Alex-Tideman thanks for catching this! I've opted to apply the filters/sorts when the user clicks on the `Workflows` or logo nav links  **while** they are on the `Recent Workflows` page. If they click away (e.g. to `Schedules`) and then click back to `Workflows` only the persisted filters will be applied.

Reply:
I haven't used afterUpdate before, just calling out to check if this could get called immediately after the onMount that has very similar code.

Reply:
I think there's some wonkiness here. Sometime when I click the nav icon, i get the initial queries, but then a couple of seconds later I get a bunch more query api calls

Reply:
> this could get called immediately after the onMount that has very similar code.

I added that conditional (`if (!query)`) to check if there already is a query so that the url isn't updated again, but the `afterUpdate` is being called after `onMount`.

> couple of seconds later I get a bunch more query api calls
There will be at least two api calls (one without the query and one with). Maybe some room for improvement there... Are you seeing additional ones? And are they related to `count`?